jumpscale/packages/admin/actors/admin.py

```diff
@@ -0,0 +1,72 @@
+from jumpscale.servers.gedis.baseactor import BaseActor, actor_method
+from jumpscale.god import j
+from jumpscale.core.exceptions import JSException
+
+explorers = {"main": "explorer.grid.tf", "testnet": "explorer.testnet.grid.tf"}
```

Comment:
what about devnet?

Reply:
Devnet is done manually not from admin dashboard